.github/workflows/ci-test.yml

```diff
@@ -648,6 +648,11 @@ jobs:
         env:
```

Comment:
are we building image from codebase?

Reply:
Yes this image is built from the codebase (not fetched from dockerhub)

Reply:
when we will switch to use docker image from doickerhub?

Reply:
The intent for this particular test is to always test against a newly built image (once branch is cut we'll also test this suite built from release branch against a newly built image) we'll need to think a bit more about how to test new clients against older servers.